### DISCUSSION TITLE
Duda practicas, basica.

### DISCUSSION THREAD

**Discussion by ArmandoRuiz13** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#57] [General]
Sobre este caso de estudio de abogados vamos a hacer futura practica o solo lo vamos a desechar, funcionando solo como aprendizaje, y si es el caso de que se siga usando. Se puede hacer de un proyecto propio?

**Comment by Ingrzs** [comment]
El mc digo que es opcional como autoevaluación, y tu decides si quieres hacerlo como proyecto o no.

**Reply by ArmandoRuiz13** [reply]
Gracias

**Comment by ApplesOranges** [comment]
Si, tú decides de qué proyecto quieres hacer la práctica.

**Reply by ArmandoRuiz13** [reply]
Gracias
